### PR TITLE
Update view-and-investigate-traces.md

### PR DESCRIPTION
**Pull request by pbrzoska-sumo**
corrected dashboard panel title

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/apm/traces/view-and-investigate-traces.md [hunk 1]
```diff
@@ -126,8 +126,8 @@ You can add Trace Duration Breakdown Chart as a dashboard panel to a new or exis
 
 1. Go to **+ New** > **Dashboard (New)**.
 1. Click the **Traces** panel type.
-1. Add the required trace query filters
-1. Under **Visual Settings** > **Chart type**, select **Trace Duration Breakdown Chart**.
+1. Add the required trace query filters representing similar traces of the same transaction.
+1. Under **Visual Settings** > **Chart type**, select **Trace Query Visualisations**.
```
Comment (JainM6) [inline docs/apm/traces/view-and-investigate-traces.md:130]:
```suggestion
1. Under **Visual Settings** > **Chart type**, select **Trace Query Visualizations**.
```

Comment (JainM6) [reply docs/apm/traces/view-and-investigate-traces.md:130]: Need a minor fix here - US English usage.

**Review by JainM6 — DISMISSED** [review]
Minor fix

**Review by kimsauce — APPROVED** [review]

**Review by JainM6 — APPROVED** [review]
LTGM

**Pull request merged**